lib/engine.js

```diff
@@ -408,15 +408,21 @@ class GameEngine {
   }
 
   exportState() {
-    return global.btoa(JSON.stringify({
+    const dump = JSON.stringify({
       state: this.initialState,
       events: this.eventsByTime,
-    }));
+    });
+    if (global.btoa) {
+      return global.btoa(dump);
+    }
+    return dump;
   }
 
   importState(dump) {
-    const imported = JSON.parse(global.atob(dump));
-
+    if (global.atob) {
```

Comment:
i didn't know that `btoa` and `atob` are only available on browser environment and not on node.js. Weird...